### ISSUE TITLE
Home page notification to pin the site to their browser home

### ISSUE THREAD

**Issue by rphovley** [CodeClimbersIO/cli#66]
When the user visits localhost:14400 and until they dismiss it or have pinned the site to their browser home, present a banner along the top to have the user to add the site to their home page with a button that does it for them in the browser they are viewing the app in.

**Comment by Tanner-Scadden** [comment]
@rphovley Are you thinking like a PWA or a bookmark in the browser?

**Comment by rphovley** [comment]
@Tanner-Scadden, sorry, I didn't look into this enough. I wanted to emulate the behavior of something like daily.dev when you visit their marketing site. Didn't realize that it was a chrome extension. We are going to make a chrome extension, or rather fork the wakatime one and give it a facelift for our stuff. I'd imagine this ticket would be added to the browser extensions we are making

**Comment by Tanner-Scadden** [comment]
@rphovley Np! Got to learn some stuff when I played with this for an hour or so :) 

Are you thinking something like this? 
<img width="1369" alt="Screenshot 2024-08-03 at 4 16 37 PM" src="https://github.com/user-attachments/assets/6c3a8f1d-47f1-401b-a08c-537a6c73a7aa">

**Comment by rphovley** [comment]
Yeah! Let's try that to start. If it's too much for people to do a modal, we can do a banner online with the other content instead
